Source/config/DBFuncs.js

```diff
@@ -0,0 +1,45 @@
+var MongoClient = require('mongodb').MongoClient;
+var url = "mongodb://localhost:27017/";
+var Building = require('../app/models/building-schema');
+var DataEntry = require('../app/models/data-entry-schema');
+var mongoose = require("mongoose");
+mongoose.createConnection("mongodb://localhost:27017/");
+var exports = module.exports = {};
+
+exports.addBuildingToDatabase = (entry) => {
+  Building.findOne({name: entry.building.name}, function (err, docs) {
+        if(docs === null){ // ensure building doesn't exist
```

Comment:
im still curious why this is not in routes. 

Why export it as module?

Reply:
I am migrating it over to routes right now. As to why I originally had it in modules, I have no idea.

Reply:
I mean Im not opposed to doing is this way, seems to marginalize the functionality, just curious to reasoning and if it would create async issues